core/vm/logger_test.go

```diff
@@ -41,9 +42,40 @@ func (d *dummyContractRef) SetBalance(*big.Int)        {}
 func (d *dummyContractRef) SetNonce(uint64)            {}
 func (d *dummyContractRef) Balance() *big.Int          { return new(big.Int) }
 
+type dummyStatedb struct {
```

Comment:
If most of the methods won't be called, you can just embed `state.StateDB` in this struct and override only `GetRefund`.